packages/components/src/entities/SamplesTabbedGridPanel.spec.tsx

```diff
@@ -1,16 +1,18 @@
 import React from 'react';
+import { act } from 'react-dom/test-utils';
 import { fromJS } from 'immutable';
 import { ReactWrapper } from 'enzyme';
 import { AuditBehaviorTypes } from '@labkey/api';
 
-import { mountWithAppServerContext } from '../internal/testHelpers';
+import { mountWithAppServerContext, waitForLifecycle } from '../internal/testHelpers';
 
 import { TEST_USER_READER, TEST_USER_STORAGE_DESIGNER } from '../internal/userFixtures';
 
 import { SchemaQuery } from '../public/SchemaQuery';
 import { QueryInfo } from '../public/QueryInfo';
 import { makeTestActions, makeTestQueryModel } from '../public/QueryModel/testUtils';
 import { TabbedGridPanel } from '../public/QueryModel/TabbedGridPanel';
+import * as sampleUtils from '../internal/components/samples/utils';
```

Comment:
import *? Is this import used or can it be removed?

Reply:
removed